my-site.rb

```diff
@@ -0,0 +1,32 @@
+require "sinatra"
+require "sinatra/reloader"
+require "./lib/scoring"
+
+class MySite < Sinatra::Base
+  register Sinatra::Reloader
+
+  get "/" do
+    erb :score
+  end
+
+  post "/" do
+  	@word = params[:word]
+  	@score = Scrabble::Scrabble.score(@word)
+
+  	erb :score
+  end
+
+  get "/multi_score" do
+  	@hash_of_words_and_scores = {}
+    erb :multi_score
+  end
+
+  post "/multi_score" do
+    @words = params[:words]
+    @word_array = Scrabble::Scrabble.create_word_array(@words)
```

Comment:
Since `@word_array` is only used to then be passed into the `score_words` method - I don't think that it should be included in your controller method. The case in which we would want it in the controller method is really only if it is needed in your multi_score view.